modules/launchd/default.nix

```diff
@@ -0,0 +1,179 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+  inherit (lib.generators) toPlist;
+  inherit (hm.dag) entryAfter;
+
+  cfg = config.launchd;
+  labelPrefix = "org.nixos";
```

Comment:
I wonder whether there could be a collision with nix-darwin's launchd services since it uses the same prefix.

Reply:
Looking at nix-darwin's activation script, I guess it would.

https://github.com/LnL7/nix-darwin/blob/44da835ac40dab5fd231298b59d83487382d2fab/modules/system/launchd.nix#L45

I changed the prefix to `org.nixos.home` to avoid collisions.